src/logic/relation.lean

```diff
@@ -194,6 +200,9 @@ begin
   case refl_trans_gen.tail : c d hbc hcd hac { exact hac.tail hcd }
 end
 
+instance : is_trans α (refl_trans_gen r) :=
+⟨@trans α r⟩
```

Comment:
I think it would make sense to put this next to `lemma transitive_refl_trans_gen : transitive (refl_trans_gen r)` below